bluepysnap/spike_report.py

```diff
@@ -93,33 +94,79 @@ def get(self, group=None, t_start=None, t_stop=None):
         """Fetch spikes from the report.
 
         Args:
-            group (int/list/np.array/dict): Get spikes filtered by group. See NodePopulation.
+            group (None/int/list/np.array/dict): Get spikes filtered by group. See NodePopulation.
             t_start (float): Include only spikes occurring after this time.
```

Comment:
Same comment as before about inclusion/exclusion of bounds.

Reply:
done